### ISSUE TITLE
Unable to create workspace nx v16.8.1 NestJs

### ISSUE THREAD

**Issue by parikshitsharma717** [nrwl/nx#19274]
### Current Behavior

When I run npx create-nx-workspace it throws an error and terminate the project setup

![image](https://github.com/nrwl/nx/assets/23037609/d9da2fd1-f468-4aa3-a5a6-7ebce1889cae)

### Expected Behavior

It should setup integrated monorepo with nestjs framework.

### Steps to Reproduce

1. Run npx create-workspace <project name>
2. select node as stack
3. select nest as framework
4. select integrated monrepo.
5. select yes to generate dockerfile
6. For distributed caching select No

### Nx Report

```shell
Node   : 18.13.0
   OS     : win32-x64
   npm    : 8.19.3

   nx (global)    : 16.8.1
   nx             : 16.8.1
   @nx/js         : 16.8.1
   @nx/jest       : 16.8.1
   @nx/linter     : 16.8.1
   @nx/workspace  : 16.8.1
   @nx/devkit     : 16.8.1
   @nx/node       : 16.8.1
   @nrwl/tao      : 16.8.1
   typescript     : 5.1.6
```

### Failure Logs

```shell
>  NX   Failed to create a workspace.

   Error: npm WARN deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
```

### Package Manager Version

8.19.3

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by ratatoeskr666** [comment]
I have the same issue but for a workspace with an angular project (NX 16.10.0)

**Comment by tekmi** [comment]
Same here. My first NX monorepo project and boom!

```cli
npx create-nx-workspace@latest test --preset=angular-monorepo
Need to install the following packages:
  create-nx-workspace@16.10.0
Ok to proceed? (y) 

 >  NX   Let's create a new workspace [https://nx.dev/getting-started/intro]

✔ Application name · diagonal-angular
✔ Default stylesheet format · scss
✔ Test runner to use for end to end (E2E) tests · cypress
✔ Would you like to use Standalone Components in your application? · Yes
✔ Would you like to add routing? · No
✔ Enable distributed caching to make your CI faster · No

 >  NX   Creating your v16.10.0 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

  To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

✔ Installing dependencies with npm
✖ Creating your workspace in test

 >  NX   Failed to create a workspace.

   Error: npm WARN deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
```

**Comment by parikshitsharma717** [comment]
BTW this issue has been resolved for me in v17.0.0

**Comment by mandarini** [comment]
@parikshitsharma717 thanks for filing an issue. Indeed that issue is solved in Nx 17! Please retry with latest version, and if issue exists please reply here and I can reopen this for you!

```
npx create-nx-workspace@latest sdfgs

 >  NX   Let's create a new workspace [https://nx.dev/getting-started/intro]

✔ Which stack do you want to use? · node
✔ What framework should be used? · nest
✔ Integrated monorepo, or standalone project? · integrated
✔ Application name · sdfgs
✔ Would you like to generate a Dockerfile? [https://docs.docker.com/] · Yes
✔ Enable distributed caching to make your CI faster · Yes

 >  NX   Creating your v17.0.1 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

✔ Installing dependencies with npm
✔ Successfully created the workspace: sdfgs.
✔ NxCloud has been set up successfully

 >  NX   Distributed caching via Nx Cloud has been enabled

   In addition to the caching, Nx Cloud provides config-free distributed execution,
   UI for viewing complex runs and GitHub integration. Learn more at https://nx.app

   Your workspace is currently unclaimed. Run details from unclaimed workspaces can be viewed on cloud.nx.app by anyone
   with the link. Claim your workspace at the following link to restrict access.

   https://cloud.nx.app/orgs/workspace-setup?accessToken=NTM1MmIyYTYtYTYxOC00MTQzLTliMTgtMGNmYWRhYmM5Y2M0fHJlYWQtd3JpdGU=

```
